pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -73,7 +73,7 @@ Comment feedback provided to students | Collected if Teacher decides to give wri
 <br>
 \* A Teacher account on Code.org has all the functionality of a Student account, and as a result the data collected and stored for a Teacher account is a superset of the data stored for a Student account.
 
-\*\* Code.org partners with U.S.-based nonprofit professional learning organizations, our “Regional Partners,” as well as similar organizations outside the U.S., our "International Partners." We share limited information (as outlined in this privacy policy, and never including Student Personal Information) with these partners to allow them to run local professional development workshops and support teachers and schools in their region. Each Regional Partner signs an Agreement with Code.org that requires them to comply with this privacy policy and to treat any information received through the partnership as confidential.
+\*\* Code.org partners with U.S.-based nonprofit professional learning organizations, our “Regional Partners,” as well as similar organizations outside the U.S., our "International Partners", collectively our "Local Partners". We share limited information (as outlined in this privacy policy, and never including Student Personal Information) with these partners to allow them to run local professional development workshops and support teachers and schools in their region. Each Local Partner signs an Agreement with Code.org that requires them to comply with this privacy policy and to treat any information received through the partnership as confidential.
```

Comment:
I had to read three times to understand that it was RP + IP == LP but this is not my specialty.